### PR TITLE
git remote org name

### PR DESCRIPTION
**Pull request by jayvdb**
Closes https://github.com/coala/community/issues/26

### CODE REVIEW

File: activity/scraper.py [hunk 1]
```diff
@@ -129,20 +133,20 @@ def get_data(self):
...
     # URL to grab all issues from
     issues_url = 'http://' + org_name + '.github.io/gh-board/issues.json'
 
     content = requests.get(issues_url)
-    content.raise_for_status()
-    parsed_json = content.json()
+    try:
+        parsed_json = content.json()
+    except Exception:
+        return '{}'
```
Comment (andrewda) [inline activity/scraper.py]: Should this be returned as an `HttpResponse`?

File: community/git.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    global _config
+    if not _config:
+        _config = GitConfigParser(GIT_CONFIG)
+    return _config
+
+
+def get_config_remote(name='origin'):
+    config = get_config()
+
+    print(config.sections())
```
Comment (andrewda) [inline community/git.py]: Is there a reason for this to be printing or is this just debug code?

File: community/git.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    if has_remote:
+        raise KeyError('No git remote called "%s"' % name)
+
+    raise KeyError('No git remotes found')
+
+
+def get_remote_url():
+    url = os.environ.get('REPOSITORY_URL')
+    if not url:
```
Comment (andrewda) [inline community/git.py]: `os.environ.get` returns None if it doesn't exist, so `if url is None:` might be better.

File: community/git.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        remote = get_config_remote()
+        url = remote[0][1]
+
+    try:
+        url = giturlparse.parse(url)
+    except Exception:
+        url = giturlparse.parse(url + '.git')
```
Comment (andrewda) [inline community/git.py]: Does this need to be `try/except`ed as well?

File: community/git.py (hunk 2 above)
Comment (jayvdb) [reply community/git.py]: whoops!

File: community/git.py (hunk 3 above)
Comment (jayvdb) [reply community/git.py]: but if someone has done `REPOSITORY_URL=''` ... :P

File: community/git.py (hunk 4 above)
Comment (jayvdb) [reply community/git.py]: No, this is a fallback to workaround https://github.com/IonicaBizau/git-url-parse/issues/35

File: community/urls.py [hunk 5]
```diff
@@ -30,16 +31,22 @@ def get_index():
         distill_func=get_index,
         distill_file='activity/index.html',
     ),
+    distill_url(
+        r'twitter/', twitter_index,
+        name='twitter',
+        distill_func=get_index,
+        distill_file='twitter/index.html',
+    ),
```
Comment (andrewda) [inline community/urls.py]: Why is this being moved?

Comment (andrewda) [inline community/urls.py]: This shouldn't get moved.

File: community/git.py [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+def get_remote_url():
+    url = os.environ.get('REPOSITORY_URL')
+    if not url:
+        remote = get_config_remote()
+        url = remote[0][1]
+
+    try:
+        url = giturlparse.parse(url)
+    except Exception:
```
Comment (andrewda) [inline community/git.py]: Only `except` necessary exceptions

File: activity/scraper.py [hunk 7]
```diff
@@ -129,20 +133,20 @@ def get_data(self):
...
 
     # URL to grab all issues from
     issues_url = 'http://' + org_name + '.github.io/gh-board/issues.json'
 
     content = requests.get(issues_url)
-    content.raise_for_status()
-    parsed_json = content.json()
+    try:
+        parsed_json = content.json()
+    except Exception:
```
Comment (andrewda) [inline activity/scraper.py]: `except` only necessary exceptions.

**Comment by yukiisbored** [comment]
ack https://github.com/coala/community/commit/10d675359e27846dc21728af88ee026f10cbe893

**Comment by yukiisbored** [comment]
ack https://github.com/coala/community/commit/59754fa0e61afe7f9b43e1b85f15aac9e7082e59

**Comment by yukiisbored** [comment]
ack https://github.com/coala/community/commit/13543868dd89c93a8dd337f1ac9365c4bad18c4d

**Comment by yukiisbored** [comment]
@blazeu Can you review https://github.com/coala/community/commit/3ef530ff4ac56a8673346baa05c963001d3ad358 ?

**Comment by jayvdb** [comment]
Preview build : https://5a34e022a6188f3fc2f4bcc9--gracious-perlman-3f3e04.netlify.com/

File: community/git.py (hunk 4 above)
Comment (jayvdb) [reply community/git.py]: No, the try/except was to workaround https://github.com/retr0h/git-url-parse/issues/2

**Comment by yukiisbored** [comment]
ack https://github.com/coala/community/pull/48/commits/3ef530ff4ac56a8673346baa05c963001d3ad358

**Pull request merged**